etc/migration/10.3-to-11.0/migration.cql3

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "Principals" ADD "emailChange" text;
+ALTER TABLE "Principals" ADD "emailVerified" boolean;
```

Comment:
I feel pretty strongly that we should try and avoid dropping more and more properties into the user profile. At the least I don't think we need `emailVerified`. If we don't need to show a "remember to verify your updated email!" on the main UI all the time, then we should avoid putting in the `emailChange` property as well.